src/content/concepts/index.mdx

```diff
@@ -19,9 +19,10 @@ contributors:
   - Jalitha
   - muhmushtaha
   - chenxsan
+  - RyanGreyling2
 ---
 
-At its core, **webpack** is a _static module bundler_ for modern JavaScript applications. When webpack processes your application, it internally builds a [dependency graph](/concepts/dependency-graph/) which maps every module your project needs and generates one or more _bundles_.
+At its core, **webpack** is a _static module bundler_ for modern JavaScript applications. When webpack processes your application, it internally builds a [dependency graph](/concepts/dependency-graph/) from one or more _entry points_ and then combines every module your project needs into one or more _bundles_, static assets to serve your content from.
```

Comment:
```suggestion
At its core, **webpack** is a _static module bundler_ for modern JavaScript applications. When webpack processes your application, it internally builds a [dependency graph](/concepts/dependency-graph/) from one or more _entry points_ and then combines every module your project needs into one or more static assets to serve your content from.
```
I think we can just remove the `bundles`.

Reply:
I would actually prefer to keep _bundles_ there. _bundles_ is used many times throughout the documentation, and I think that paragraph would be the perfect place to define what they are.

Reply:
The reason I want to reword this is because "combines every module your project needs into one or more _bundles_, static assets to serve your content from." seems confusing to a non-native English speaker. I don't really know how to read a sentence like this. A conjunction like `i.e.,` or `namely` would be really helpful for me:

> combines every module your project needs into one or more _bundles_, i.e., static assets to serve your content from.

Reply:
```suggestion
At its core, **webpack** is a _static module bundler_ for modern JavaScript applications. When webpack processes your application, it internally builds a [dependency graph](/concepts/dependency-graph/) from one or more _entry points_ and then combines every module your project needs into one or more _bundles_, i.e., static assets to serve your content from.
```
```suggestion
At its core, **webpack** is a _static module bundler_ for modern JavaScript applications. When webpack processes your application, it internally builds a [dependency graph](/concepts/dependency-graph/) from one or more _entry points_ and then combines every module your project needs into one or more _bundles_, which are static assets to serve your content from.
```
Good point. I hadn't considered readability for non-native speakers. I made two possible revisions. The first one uses your "i.e." idea and the second uses "which".